src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -1269,22 +1330,36 @@ int eventer_ssl_config(const char *key, const char *value) {
   }
   return 1;
 }
+
+static int32_t initialized = 0;
```

Comment:
Needs `volatile` here? I think `eventer_ssl_init()` could be called from different threads..

Reply:
It can't really.. We call this once, this is just protection in case a user is dumb and calls it again.